### ISSUE TITLE
Sample query designDoc

### ISSUE THREAD

**Issue by CaptainMM** [colinskow/superlogin#164]
I've successfully created .js based designDocs for superlogin but am having trouble using query indexes. Is this supported? For instance, how would we use a design doc for a simple index which Fauxton creates?

{
  "_id": "_design/1",
  "language": "query",
  "views": {
    "2": {
      "map": {
        "fields": {
          "date": "asc"
        }
      },
      "reduce": "_count",
      "options": {
        "def": {
          "fields": [
            "date"
          ]
        }
      }
    }
  }
}

**Comment by peteruithoven** [comment]
I feel like the following question of mine asks the same question or maybe is related? 
https://github.com/colinskow/superlogin/issues/116

**Comment by CaptainMM** [comment]
Yes, I've looked over your question and that's definitely the main reason I can't just simply create an index when the user logs in using pouch therefore I assume we have to do this from a designDoc or add more lenient db security? The only problem is I can't find a good example of a design doc where superlogin accepts query as the language. If we can do the same thing in js I'd gladly go that route. The only example I see is the user-design.js doc.

**Comment by peteruithoven** [comment]
Superlogin [uses](https://github.com/colinskow/superlogin/blob/master/lib/dbauth/index.js#L79) [pouch-seed-design](https://github.com/colinskow/pouchdb-seed-design). I don't see any mention of support for the query language. I think we might need to start an issue and/or pull request there.

For examples of design docs I would check the CouchDB documentation:
http://docs.couchdb.org/en/2.0.0/intro/tour.html?highlight=design%20doc#running-a-query-using-mapreduce
http://docs.couchdb.org/en/2.0.0/couchapp/views/intro.html?highlight=design%20docs

If I understand it correctly the new query language only makes it easier, it doesn't add functionality?
